.eslintignore

```diff
@@ -149,9 +149,13 @@ packages/labs/scoped-registry-mixin/scoped-registry-mixin.*
 packages/labs/ssr/demo/
 packages/labs/ssr/lib/
 packages/labs/ssr/node_modules/
-packages/labs/ssr/test/
+packages/labs/ssr/test/**/*.js
```

Comment:
Are these changes needed? Maybe they belong in another PR that adds tests? Same for `.prettierignore`.

Reply:
This PR does add tests. Some of your comments were on older version files though, so maybe you didn't see them?

Reply:
Oops yeah I missed them somehow.